pkg/sdk/event_tables_def.go

```diff
@@ -0,0 +1,148 @@
+		OptionalBooleanAssignment("CHANGE_TRACKING", g.ParameterOptions()).
+		OptionalTextAssignment("DEFAULT_DDL_COLLATION", g.ParameterOptions().SingleQuotes()).
+		OptionalSQL("COPY GRANTS").
+		OptionalTextAssignment("COMMENT", g.ParameterOptions().SingleQuotes()).
+		PredefinedQueryStructField("RowAccessPolicy", "*RowAccessPolicy", g.KeywordOptions()).
+		OptionalTags().WithValidation(g.ValidIdentifier, "name"),
+).ShowOperation(
+	"https://docs.snowflake.com/en/sql-reference/sql/show-event-tables",
+	g.DbStruct("eventTableRow").
+		Field("created_on", "string").
+		Field("name", "string").
+		Field("database_name", "string").
+		Field("schema_name", "string").
+		Field("owner", "string").
+		Field("comment", "string").
+		Field("owner_role_type", "string").
+		Field("change_tracking", "string"),
+	g.PlainStruct("EventTable").
+		Field("CreatedOn", "string").
+		Field("Name", "string").
+		Field("DatabaseName", "string").
+		Field("SchemaName", "string").
+		Field("Owner", "string").
+		Field("Comment", "string").
+		Field("OwnerRoleType", "string").
+		Field("ChangeTracking", "bool"),
+	g.NewQueryStruct("ShowFunctions").
+		Show().
+		SQL("EVENT TABLES").
+		OptionalLike().
+		OptionalIn().
+		OptionalTextAssignment("STARTS WITH", g.ParameterOptions().SingleQuotes().NoEquals()).
+		OptionalNumberAssignment("LIMIT", g.ParameterOptions()).
+		OptionalTextAssignment("FROM", g.ParameterOptions().SingleQuotes().NoEquals()),
+).DescribeOperation(
+	g.DescriptionMappingKindSingleValue,
+	"https://docs.snowflake.com/en/sql-reference/sql/describe-event-table",
+	g.DbStruct("eventTableDetailsRow").
+		Field("name", "string").
+		Field("kind", "string").
+		Field("comment", "string"),
+	g.PlainStruct("EventTableDetails").
+		Field("Name", "string").
+		Field("Kind", "string").
+		Field("Comment", "string"),
+	g.NewQueryStruct("DescribeEventTable").
+		Describe().
+		SQL("EVENT TABLE").
+		Name().
+		WithValidation(g.ValidIdentifier, "name"),
+).AlterOperation(
+	"https://docs.snowflake.com/en/sql-reference/sql/alter-event-table",
+	g.NewQueryStruct("AlterEventTable").
+		Alter().
+		SQL("TABLE").
+		IfNotExists().
+		Name().
+		OptionalQueryStructField(
+			"Set",
+			eventTableSet,
+			g.KeywordOptions().SQL("SET"),
+		).
+		OptionalQueryStructField(
+			"Unset",
+			eventTableUnset,
+			g.KeywordOptions().SQL("UNSET"),
+		).
+		PredefinedQueryStructField("AddRowAccessPolicy", "*RowAccessPolicy", g.KeywordOptions().SQL("ADD")).
+		OptionalQueryStructField(
+			"DropRowAccessPolicy",
+			eventTableDropRowAccessPolicy,
+			g.KeywordOptions().SQL("DROP"),
+		).
+		OptionalSQL("DROP ALL ROW ACCESS POLICIES").
+		OptionalQueryStructField(
+			"ClusteringAction",
+			eventTableClusteringAction,
+			g.KeywordOptions(),
+		).
+		OptionalQueryStructField(
+			"SearchOptimizationAction",
+			eventTableSearchOptimizationAction,
+			g.KeywordOptions(),
+		).
+		PredefinedQueryStructField("SetTags", "[]TagAssociation", g.KeywordOptions().SQL("SET TAG")).
```

Comment:
We have `OptionalSetTags()` and `OptionalUnsetTags()`

Reply:
fixed